common/utils/utils.go

```diff
@@ -73,6 +74,13 @@ func GetExternalIPAddress() (externalIP string, err error) {
 	return string(body), nil
 }
 
+// GetHashFromString generate sha256 hash from a given string
+func GetHashFromString(inputString string) string {
```

Comment:
there is already existing function for this in this file named `Sha3256hash(msg []byte) ([]byte, error)`